### PR TITLE
LEARN-POEMS-T-26 Add GitHub Actions test job

### PR DESCRIPTION
**Pull request by poymanov**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ master, develop ]
```
Comment (ekut) [inline .github/workflows/ci.yml]: We don't use the `master` branch, the `main` instead of it.

**Review by ekut — CHANGES_REQUESTED** [review]
Нужно малюсенькое изменение.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (poymanov) [reply .github/workflows/ci.yml]: Done

**Review by eljump — APPROVED** [review]
Good Job)

**Review by ekut — APPROVED** [review]

**Pull request merged**